oddbird/MetaDeploy
# 🛢 Add consistent URL ("/latest/") for most recent product version.

![](https://source.unsplash.com/featured/?cute,animal&210820)

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Navigate around site; see `/latest/` in URL
